### PR TITLE
update module

### PR DESCRIPTION
**Pull request by milestorme**
add travis (needs activation : https://travis-ci.org/azerothcore/mod-solocraft ) update module, cleanup

### CODE REVIEW

**Comment by FrancescoBorzi** [comment]
travis activated

File: LICENSE [hunk 1]
```diff
@@ -1,23 +1,21 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (BarbzYHOOL) [inline LICENSE]: ?

File: src/Solocraft.cpp [hunk 2]
```diff
@@ -184,9 +138,9 @@ class solocraft_player_instance_handler : public PlayerScript {
             // InstanceScript *instanceScript = instanceMap->GetInstanceScript();
 
             // Announce to player
-            std::ostringstream ss;
-            ss << "|cffFF0000[SoloCraft] |cffFF8000" << player->GetName() << " entered %s - # of Players: %d - Difficulty Offset: %d.";
-            ChatHandler(player->GetSession()).PSendSysMessage(ss.str().c_str(), map->GetMapName(), numInGroup, difficulty);
+           // std::ostringstream ss;
+           // ss << "|cffFF0000[SoloCraft] |cffFF8000" << player->GetName() << " entered %s - # of Players: %d - Difficulty Offset: %d.";
+           // ChatHandler(player->GetSession()).PSendSysMessage(ss.str().c_str(), map->GetMapName(), numInGroup, difficulty);
```
Comment (BarbzYHOOL) [inline src/Solocraft.cpp]: why removed? why not announce but only to group or to the player only (which was the case already no?)?

Comment (milestorme) [reply src/Solocraft.cpp]: My bad that was a personal setting of mine I’ll adjust that

File: LICENSE (hunk 1 above)
Comment (milestorme) [reply LICENSE]: I used another module as a template , probably used the wrong one should have used Skelton mod , I’ll fix it 👍

Comment (BarbzYHOOL) [reply LICENSE]: I don't know much about licenses but it should stay as original, should revert the license files (or lines)

Comment (milestorme) [reply LICENSE]: there i grabbed the original for this module from ac repo

**Comment by BarbzYHOOL** [comment]
hey milestorme, you could run in each of your module directory the bash script

you open with git bash the directory and you execute `setup_git_commit_template.sh`

Example:

`bash setup_git_commit_template.sh`
or
`./setup_git_commit_template.sh`

It will help when committing

**Comment by BarbzYHOOL** [comment]
merged, thank you

**Pull request merged**